### PR TITLE
Accessibility and policies page

### PR DESCRIPTION
**Pull request by skade**
Rebrushed the accessibility page. We now have so many things in the policies area that I collapsed them to an overview page.

### CODE REVIEW

File: _data/header_menu.yml [hunk 1]
```diff
@@ -2,3 +2,5 @@
   link: https://blog.rustfest.eu/
 - title: Past Events
   link: https://blog.rustfest.eu/past_events/
+- title: "Info & Policies"
+  link: https://rome.rustfest.eu/policies/
```
Comment (badboy) [inline _data/header_menu.yml]: Use a relative link here.
Also it seems like it is /info/

File: policies.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: blue_page
+permalink: /info/
```
Comment (badboy) [inline policies.md]: This is located under /info/. Any objection to renaming the file to info.md as well? (Or change the url to policies) to keep it consistent?

File: policies.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+title: Information & Policies
+---
+
+RustFest documents all their policies openly. They can be found here:
+
+* [Accessibility](/accessibility)
+* [Code of Conduct](/coc)
```
Comment (badboy) [inline policies.md]: CoC is at /code-of-conduct/

File: policies.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    * We follow the PyCon [Attendee](https://us.pycon.org/2018/about/code-of-conduct/attendee-procedure/) and [Staff](https://us.pycon.org/2018/about/code-of-conduct/staff-procedure/) procedure for reporting
+* [Parents and nursing mothers](/parents)
+* [CFP submission and reimbursement policies](https://cfp.rustfest.eu)
```
Comment (badboy) [inline policies.md]: Do we have/will we have any more direct link for this?

File: policies.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+* [Diversity tickets and conditions](https://diversitytickets.org)
```
Comment (badboy) [inline policies.md]: Should we have a general "Something unclear? Mail us" here?

File: accessibility.md [hunk 6]
```diff
@@ -4,24 +4,24 @@ permalink: /accessibility/
...
 RustFest is an inclusive conference and as such strives to be accessible to everyone. This includes:
 
 * Offering significantly discounted tickets for concession holders.
-* Ensuring our venues are accessible by wheelchair.
-* Making sure that maps and orientation material is available.
-* Providing clear signage to, from and through the building.
+* Ensuring all our venues are accessible by wheelchair.
+* Live transcription of all talks.
+    - Appropriate measures can be taken for the workshop day and the impl days. Please get in touch.
+* Makeing sure that maps and orientation material is available.
```
Comment (badboy) [inline accessibility.md]:
```patch
- Makeing
+ Making
```

File: accessibility.md [hunk 7]
```diff
@@ -4,24 +4,24 @@ permalink: /accessibility/
...
+* Providing clear signage to, from and through the building
 * Providing quiet spaces for those with sensory issues.
 * Providing accessibility for conference sessions.
-* Providing complimentary tickets for assistants for disabled participants.
+* Providing complimentary tickets for assistants for those with disabilities.
+* Access to non-gendered toilets.
```
Comment (badboy) [inline accessibility.md:18]: At this point, can we already guarantuee that for the main venue?

File: accessibility.md [hunk 8]
```diff
@@ -4,24 +4,24 @@ permalink: /accessibility/
...
 * Serving a wide range of food and special requests.
-* We provide non-gendered toilets.
-* Sadly, only guide animals are allowed at the venue.
-
-If you're not sure whether the concession rate should apply to you, please do get in touch.
+* An all-around help hotline during the time of the conference.
+* Providing 5 tickets specifically through [diversityticket.org](https://diversitytickets.org).
```
Comment (badboy) [inline accessibility.md]: @flaki What do we budget for? 5?

**Review by badboy — CHANGES_REQUESTED** [review]

File: accessibility.md (hunk 8 above)
Comment (skade) [reply accessibility.md]: We discussed that Tuesday, it's taken from the the general pool, so it's cost-neutral.

Comment (badboy) [reply accessibility.md]: Alright.

File: policies.md (hunk 4 above)
Comment (skade) [reply policies.md]: Not yet, the event is still private. I'll link to paris for now

**Comment by skade** [comment]
Fixed all comments.

**Review by badboy — APPROVED** [review]

File: accessibility.md (hunk 7 above)
Comment (skade) [reply accessibility.md:18]: I'm not sure, I'll rephrase

Comment (skade) [reply accessibility.md:18]: https://github.com/RustFestEU/rome.rustfest.eu/pull/2/commits/89379ce871c15c2b20bec27c87fc7aa59817394d

File: accessibility.md [hunk 9]
```diff
@@ -16,6 +16,8 @@ RustFest is an inclusive conference and as such strives to be accessible to ever
 * Providing accessibility for conference sessions.
 * Providing complimentary tickets for assistants for those with disabilities.
 * Access to non-gendered toilets.
+    - This is sometimes subject to the venues rules, but we do our best to ensure this.
+    - If not, we expect people not to complain about others choice of toilet.
```
Comment (apiraino) [inline accessibility.md]: @skade not sure about this sentence

You mean to say that "if not, we hope attendees will understand that this is out of reach for us"? ¯\_(ツ)_/¯

File: accessibility.md [hunk 10]
```diff
@@ -4,24 +4,26 @@ permalink: /accessibility/
...
 * Providing quiet spaces for those with sensory issues.
 * Providing accessibility for conference sessions.
-* Providing complimentary tickets for assistants for disabled participants.
+* Providing complimentary tickets for assistants for those with disabilities.
+* Access to non-gendered toilets.
+    - This is sometimes subject to the venues rules, but we do our best to ensure this.
+    - If not, we expect people not to complain about others choice of toilet.
 * Serving a wide range of food and special requests.
-* We provide non-gendered toilets.
-* Sadly, only guide animals are allowed at the venue.
```
Comment (apiraino) [inline accessibility.md]: Discuss whether to revert this line into the accessibility policies

File: accessibility.md (hunk 9 above)
Comment (skade) [reply accessibility.md]: Yeah, this is ambigious. It's "expect" in the sense of "we want you to and will react if you don't".

Comment (skade) [reply accessibility.md]: I removed the two for now, we can further refine later.

**Review by apiraino — APPROVED** [review]

**Pull request merged**